packages/components/src/themes.ts

```diff
@@ -33,6 +33,31 @@ const light = createTheme({
   brownHover: tokens.color['brown-60'],
   beige: tokens.color['beige-50'],
   beigeHover: tokens.color['beige-60'],
+  placeHolderColor: tokens.color['neutral-40'],
+  placeHolderColorBlurred: tokens.color['neutral-80-opa-40'],
+  iconButtonBackground: tokens.color['neutral-10'],
+  iconButtonBackgroundHover: tokens.color['neutral-20'],
+  iconButtonBackgroundSelected: tokens.color['neutral-20'],
+  iconButtonBorderSelected: tokens.color['neutral-30'],
+  iconButtonBackgroundBlurred: tokens.color['neutral-80-opa-5'],
+  iconButtonBackgroundBlurredHover: tokens.color['neutral-80-opa-10'],
+  iconButtonBackgroundBlurredSelected: tokens.color['neutral-80-opa-10'],
+  iconButtonBorderBlurredSelected: tokens.color['neutral-80-opa-5'],
+  iconButtonOutlineBackgroundSelected: tokens.color['neutral-10'],
+  iconButtonOutlineBorder: tokens.color['neutral-20'],
+  iconButtonOutlineBorderHover: tokens.color['neutral-30'],
+  iconButtonOutlineBorderSelected: tokens.color['neutral-20'],
+  iconButtonOutBackgroundBlurredSelected: tokens.color['neutral-10'],
+  iconButtonOutlineBorderBlurred: tokens.color['neutral-80-opa-10'],
+  iconButtonOutlineBorderBlurredHover: tokens.color['neutral-80-opa-20'],
+  iconButtonOutlineBorderBlurredSelected: tokens.color['neutral-80-opa-10'],
+  iconButtonColor: tokens.color['neutral-50'],
+  iconButtonColorSelected: tokens.color['neutral-100'],
+  iconButtonColorBlurred: tokens.color['neutral-100'],
+  iconButtonColorOutline: tokens.color['neutral-50'],
+  iconButtonColorOutlineSelected: tokens.color['neutral-100'],
+  iconButtonColorOutlineBlurred: tokens.color['neutral-80-opa-70'],
+  blurBackground: tokens.color['white-70'],
```

Comment:
I need to read Tamagui's documentation, but I don't see sense in doing this. I think the colors should be just exposed directly in the theme.

Reply:
1 - Color tokens are not working (only theme tokens) for some specific styles inside variants (eg: `placeholderTextColor`).
2 - If you have different colors to use with different themes in some components you'll need to use tokens so that it uses a different color value for each theme. Otherwise how would you do it?

Reply:
I just found out that color tokens are different for every theme. This needs to be addressed with designers because that misses the whole point of tokens.